README.md

```diff
@@ -108,40 +108,6 @@ Amoro contains modules as below:
     - `amoro-spark` provides Spark connectors for Mixed format tables (use amoro-spark-runtime for a shaded version)
     - `amoro-trino` provides Trino connectors for Mixed format tables
 
-
```

Comment:
IMHO, people who find out the project first time are more interested in how to use it than in building it, so I move the build steps directly to `CONTRIBUTING`

Reply:
GitHub is the platform for developers.
So I prefer keeping the building sections int README for developers.
And I found that some other open-source projects closely related to Amoro are also doing the same，like:

- Apache Iceberg: https://github.com/apache/iceberg/blob/main/README.md
- Apache Flink: https://github.com/apache/flink/blob/main/README.md
- Apache Paimon(incubating): https://github.com/apache/incubator-paimon/blob/main/README.md

Reply:
ok，will keep the build steps in README

Reply:
ok，will keep the build steps in README